src/main/scala/scalismo/common/interpolation/ValueInterpolator.scala

```diff
@@ -51,41 +51,44 @@ trait Interpolator[@specialized(Double, Float) A] {
     var mix: A = first
     var n = 1
     for (next <- rest) {
-      mix = blend(mix, next, n.toFloat / (n + 1))
+      mix = blend(mix, next, n / (n + 1))
```

Comment:
@Andreas-Forster Nice catch. I mistakenly thought the toFloat was because of a Double to Float conversion.
I also completely agree that the ValueInterpolator should go into the numerics package. 

Regarding the naming: The other interpolators do already have more specific names (e.g. FieldInterpolator, LinearImageInterpolator, etc.) to make it more clear what their purpose is. Leaving the name of this class as Interpolator would suggest that it is somehow more general than all the others. This is clearly not true, as this interpolator has  the very narrow purpose of interpolating two values. Furthermore it will never be called directly by a user. Therefore I think keeping the current name would be confusing .

Renaming the Interpolator trait should not break any code, as this class is not used directly from user code. Even if it was, it is a good moment to rename it, as moving it to another package would anyway break the code.